src/msg/async/Stack.h

```diff
@@ -193,6 +193,7 @@ enum {
   l_msgr_send_bytes,
   l_msgr_created_connections,
   l_msgr_active_connections,
+  l_msgr_osdop_lifetime_lat,
```

Comment:
it's a little confusing to put osd op latency here, could we delegate logger in MOSDOpReply Message?